### ISSUE TITLE
SMS verification code for 2FA not appearing

### ISSUE THREAD

**Issue by gkomarnicki** [moezbhatti/qksms#1206]
### DESCRIPTION
I had a strange bug with QKSMS when trying to receive the verification code for turning on 2FA in my bit.ly account. After requesting the code, I didn't get any notification. A second time - still nothing. I even tried a third time the next day but still nothing. It was only after opening Android Messages and Pulse did I notice the phone numbers containing the verification codes, but they didn't show up in QKSMS.

Huawei Nova 2i (Mate 10 Lite), 8.0 Oreo

### STEPS
1. Request for a 2FA verification code
2. Check if the code appears in QKSMS

### EXPECTED
The 2FA verification code should appear in QKSMS, like it does in other SMS apps.

### OBSERVATIONS
The verification code doesn't show up in QKSMS for some reason.

**Comment by moezbhatti** [comment]
Duplicate of #1200
